www/contribute/wanted-ebooks.php

```diff
@@ -242,7 +242,7 @@
 		<h2>Moderate-difficulty productions</h2>
 		<ul>
 			<li>
-				<p>Gentlement Prefer Blondes by Anita Loos, transcript in progress <a href="https://www.pgdp.net/c/project.php?id=projectID5ff6baf1b5b4b">at PGDP</a>. Once the transcript is available, this may be suitable for a beginner.  (Part of the Guardian 2015 collection.)</p>
+				<p>Gentlement Prefer Blondes by Anita Loos, transcript in progress <a href="https://www.pgdp.net/c/project.php?id=projectID5ff6baf1b5b4b">at PGDP</a>. Once the transcript is available, this may be suitable for a beginner.  (Part of the Guardian 2015 collection.)
```

Comment:
We don’t need to remove the closing p.

Reply:
Yes indeed, I don't know why this is missing. Apologies!

```suggestion
				<p>Gentlement Prefer Blondes by Anita Loos, transcript in progress <a href="https://www.pgdp.net/c/project.php?id=projectID5ff6baf1b5b4b">at PGDP</a>. Once the transcript is available, this may be suitable for a beginner.  (Part of the Guardian 2015 collection.)</p>
```